src/libical/icalparser.h

```diff
@@ -317,8 +317,7 @@ LIBICAL_ICAL_EXPORT icalcomponent *icalparser_parse_string(const char *str);
  ***********************************************************************/
 
 /**
- * @brief Given a line generator function, return a single iCal content line.
- * @return Aa pointer to a single line of data or `NULL` if it reached
```

Comment:
I think we can keep the " @return A pointer to a single line of data or `NULL` if it reached"

Reply:
Definitely agree. Didn't realize I removed that. I'll fix it now.

Reply:
Actually, it's still there in the set of all changes. Not sure why it wasn't showing up in the set of commits you reviewed.

Reply:
dunno.